src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -186,8 +186,12 @@ def _print_items(items):
         _print_items(subcommands)
 
 def _get_choices_defaults_sources_str(p):
-    choice_str = '  Allowed values: {0}.'.format(', '.join(sorted([str(x) for x in p.choices]))) \
-        if p.choices else ''
+    # Convert each choice item to string and add double quotes if string contains white space
+    def _stringify(item):
+        item_str = str(item)
+        return '"{0}"'.format(item_str) if ' ' in item_str else item_str
+    choice_str = '  Allowed values: {0}.'.format(
```

Comment:
Please add a blank line between the function definition and the following statements to improve readability.